### PR TITLE
[Doc Improvement][Rate limiting doc code update][3315502]

### PR DESCRIPTION
**Pull request by v-rmvv**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/rate-limit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3f3943f13306e57965eda177021d433ee9a385c0/msteams-platform%2Fbots%2Fhow-to%2Frate-limit.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6dc468f0-7415-4b00-8e50-61f0b3a91b25) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/rate-limit.md [hunk 1]
```diff
@@ -54,9 +54,10 @@ The following code shows an example of using exponential backoff using the trans
 public class BotSdkTransientExceptionDetectionStrategy : ITransientErrorDetectionStrategy
     {
         // List of error codes to retry on
-        List<int> transientErrorStatusCodes = new List<int>() { 429 };
+        private static readonly List<int> transientErrorStatusCodes = new List<int>() { 429 };
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/rate-limit.md:57]: No changes required for this line remove : private static readonly

File: msteams-platform/bots/how-to/rate-limit.md [hunk 2]
```diff
@@ -54,9 +54,10 @@ The following code shows an example of using exponential backoff using the trans
...
 
-        public static bool IsTransient(Exception ex)
+        public bool IsTransient(Exception ex)
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/rate-limit.md:59]: This change is fine.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comment

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/rate-limit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1e131ff6bf2e0d70fb5d3d289a018e466086706/msteams-platform%2Fbots%2Fhow-to%2Frate-limit.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a44c9946-cfd9-41b0-a385-ba79dd34b126) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/rate-limit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e78b117fe41c474019e28ec8330addf2759f0d7d/msteams-platform%2Fbots%2Fhow-to%2Frate-limit.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50b600f6-4387-4360-a1cc-d46851a4ca74) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/rate-limit.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/bots/how-to/rate-limit.md]: Updated.

**Review by ChetanSharma-msft — APPROVED** [review]
looks fine.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/rate-limit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8fcf7a3f4e30dd78b1b3d55032a4773254d9587b/msteams-platform%2Fbots%2Fhow-to%2Frate-limit.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f23f6492-ea34-442f-89b6-ef4997756236) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**